packages/core/src/constants.ts

```diff
@@ -1,6 +1,5 @@
 export const InjectionSymbols = {
   Wallet: Symbol('Wallet'),
-  Indy: Symbol('Indy'),
```

Comment:
That one's a bit more tricky because we initiate it inside the agent and want to have it available across the agent. But thinking of it now I'll add it the agentConfig